README.md

```
@@ -9,7 +9,7 @@
     <img src="https://img.shields.io/badge/pydantic-v2-orange?style=flat-square" />
     <img src="https://img.shields.io/badge/pypi-aura--web-purple?style=flat-square" />
     <img src="https://img.shields.io/badge/version-0.3.0-blue?style=flat-square" />
-    <img src="https://img.shields.io/badge/tests-243%20passing-brightgreen?style=flat-square" />
+    <img src="https://img.shields.io/badge/tests-347%20passing-brightgreen?style=flat-square" />
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)